test_massive_generator.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+from generator_massive import MassiveGenerator
+
+
+class TestMassiveGenerator(unittest.TestCase):
+    @unittest.expectedFailure
+    def test_empty_massive(self):
+        MassiveGenerator([])
+
+    @unittest.expectedFailure
```

Comment:
This is absolutely counterintuitive, isn't it?

Reply:
I guess the program must check whether the number of arguments correct or not. When I run print(range()), I receive "TypeError: range expected 1 arguments, got 0", so I try to create the same behaviour for my program.